src/np_array.php

```diff
@@ -11,6 +11,28 @@ class np_array extends \ArrayObject
 {
     use Indexer, Printer, Statistics;
 
+    private $level = 0;
+    private $size = null;
+
+    public function __construct($input = [], $flags = 0, $iterator_class = "ArrayIterator", $level = 0)
```

Comment:
Maybe it will be better for using `...__construct( array $input = []...`

And you can remove if-statement from 22 line